app/views/users/show.html.haml

```diff
@@ -0,0 +1,37 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
+          %h2= "#{@user.name}'s video collections (#{@user.total_queue_items})"
+          = button_to "Follow", relationships_path(following_id: @user), class: "btn btn-default", style: "float: right;" unless @user == current_user || current_user.following?(@user)
+
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |queued_item|
+              %tr
+                %td
+                  = link_to "#{queued_item.video.title}", queued_item.video
+                %td
+                  = link_to "#{queued_item.video.category.name}", queued_item.video.category
+
+%section.user_reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %header
+        %h3= "#{@user.name}'s's Reviews (#{@user.total_reviews})"
+      %ul
+        - @user.reviews.each do |review|
+          %article.review
+            %li.row
+              .col-sm-2
+                %p
+                  = link_to "#{review.video.title}"
+                %col= "Rating: #{review.rating.to_i} / 5"
```

Comment:
Instead of this: `%col= "Rating: #{review.rating.to_i} / 5"` you can do `%col Rating: #{review.rating.to_i} / 5`